kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -499,6 +502,49 @@ object KyuubiConf {
     .checkValue(p => p == 0 || (p > 1024 && p < 65535), "Invalid Port number")
     .createWithDefault(10099)
 
+  val FRONTEND_MYSQL_BIND_HOST: ConfigEntry[Option[String]] =
+    buildConf("frontend.mysql.bind.host")
+      .doc("Hostname or IP of the machine on which to run the MySQL frontend service.")
+      .version("1.4.0")
+      .fallbackConf(FRONTEND_BIND_HOST)
+
+  val FRONTEND_MYSQL_BIND_PORT: ConfigEntry[Int] = buildConf("frontend.mysql.bind.port")
+    .doc("Port of the machine on which to run the MySQL frontend service.")
+    .version("1.4.0")
+    .intConf
+    .checkValue(p => p == 0 || (p > 1024 && p < 65535), "Invalid Port number")
+    .createWithDefault(3309)
+
+  val FRONTEND_MYSQL_NETTY_WORKER_THREADS: OptionalConfigEntry[Int] =
+    buildConf("frontend.mysql.netty.worker.threads")
+      .doc("Number of thread in the netty worker event loop of MySQL frontend service")
```

Comment:
shall we mention the default thread num is `sys.runtime.availableProcessors` in doc ?

Reply:
Actually, it should be `min(sys.runtime.availableProcessors, MAX_NETTY_THREADS)`, I will update it.